### PR TITLE
Added -j option to bnd xref command

### PR DESCRIPTION
**Pull request by pkriens**
---
 Signed-off-by: Peter Kriens <Peter.Kriens@aQute.biz>

### CODE REVIEW

**Comment by kwin** [comment]
Thanks a lot, that was quick!

File: biz.aQute.bnd/src/aQute/bnd/main/XRefCommand.java [hunk 1]
```diff
@@ -57,6 +57,8 @@ interface xrefOptions extends Options {
 		@Description("Output list of package/class names that have been referred to")
 		String referrredTo();
 
+		@Description("Include java.* packages")
+		boolean java();
```
Comment (kwin) [inline biz.aQute.bnd/src/aQute/bnd/main/XRefCommand.java:61]: Don't you need to evaluate that flag somewhere in https://github.com/bndtools/bnd/blob/d56166137d9b1e82381d205b1837998cd3fad87e/biz.aQute.bnd/src/aQute/bnd/main/XRefCommand.java#L73?

**Comment by pkriens** [comment]
Too quick, as you already discovered forgot to check it :-)

**Comment by pkriens** [comment]
@kwin if you can test it than I merge it

**Comment by kwin** [comment]
https://github.com/bndtools/bnd/blob/master/docs/_commands/xref.md needs some adjustment as well.

**Comment by pkriens** [comment]
I leave the doc up to you ... bnd help xref is always up to date

**Comment by kwin** [comment]
I just tested and it works like a charm. Thanks again.

**Pull request merged**